its/SonarCloudCallWrapper.cs

```diff
@@ -0,0 +1,112 @@
+﻿using AzureDevOpsExtension.IntegrationTests.Models;
+using Newtonsoft.Json;
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AzureDevOpsExtension.IntegrationTests
+{
+    public class SonarCloudCallWrapper
+    {
+        private string GetBase64EncodedToken()
+        {
+            return Convert.ToBase64String(ASCIIEncoding.ASCII.GetBytes(String.Format("{0}:", Environment.GetEnvironmentVariable("SC_TOKEN"))));
```

Comment:
I don't fully understand why using String.Format, can it be replaced with an interpolated string?